config/default.yml

```diff
@@ -0,0 +1,11 @@
+server:
+  host: localhost
+  apiHost: localhost
+  port: 3000
+  wdsPort: 8080
+  apiPort: 3002
+mongo:
+  protocol: mongodb
+  host: localhost
+  port: 27017
+  db: Interact
```

Comment:
I think you didn't look into the deleted config file (Ref) when you were fixing the conflicts, but next time please look into it carefully because I already fixed the wrong `port` for `wds` and add `ws` `port`.

First I suggest you to write it like this. I think it is more meaningful than just server.something

```yml
ssr:
  host: localhost
  port: 3000
wds:
  host: localhost
  port: 3001
api:
  host: localhost
  port: 3002
ws:
  host: localhost
  port: 8080
mongo:
...
```

Second, since you didn't fix the conflict properly so I think now you can not run the server or else something might goes wrong. What you need to do now is look for the `config.wsPort` I added in `api-server` and fix it.